### PR TITLE
PullRequest for Issue2255: add the 2D summing analysis when both burker and 4E is selected

### PR DESCRIPTION
**Pull request by XuetaoLiu-CLS**
#2255

### CODE REVIEW

**Comment by XuetaoLiu-CLS** [comment]
@helfrij can you review? Thx

**Comment by helfrij** [comment]
Looks good, @XuetaoLiu-CLS.

Time Estimate: 10 minutes.

File: source/acquaman/SXRMB/SXRMBEXAFSScanActionController.h [hunk 1]
```diff
@@ -34,6 +30,12 @@ protected slots:
 	/// Reimplemented for EXAFS capabilities.  Creates the scan assembler that builds all the actions used to run the scan.
 	virtual void createScanAssembler();
 
+	/// build analysis block for XRF detectors
+	void buildXRFAnalysisBlock(const QList<AMDataSource *> i0Sources);
```
Comment (dretrex) [inline source/acquaman/SXRMB/SXRMBEXAFSScanActionController.h]: Missing the & in `const QList &`.

**Comment by dretrex** [comment]
Looks good other than the one comment.
Time Estimate: 3 minutes

**Pull request merged**